README.rst

```diff
@@ -20,33 +20,68 @@ configurations remain synchronized with the state of all databases in the
 cluster. PgQ is used for buffering mutation records until they are collected by
 the relay and forwarded to their final destination.
 
+Function
```

Comment:
"Usage" to me reads like "this is how you use X", not use cases. I think either `Function` or `Overview` is OK — was just providing an alternative since "function" is a bit overloaded.